core/templates/dev/head/pages/question_editor/question_editor_directive.html

```diff
@@ -0,0 +1 @@
+<!-- This file will contain the reusable question directive html -->
```

Comment:
Make this a TODO with an instruction so that people know who to contact if they see this file and wonder why it's there.

Reply:
Done.